usagov/usagov-2021
# Usagov 622 Move Header

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-622

## Description


Created a twig file to style a top navigation menu. 
- page.html.twig calls top_nav.html.twig. 
- top_nav.html.twig holds menu that used to be in the header block. This includes the logo, search box and contact but calls menu--top-navigation.html.twig to create the top nav
- menu--top-navigation.html.twig styles the Top Navigation Menu which will be created in Drupal CMS so content can edit it. 

### Testing Instructions
- Config changes do need to be imported! 
- You should see no top nav menu
- You also will need to create a new menu called Top Navigation (machine-name "top-navigation") (This should be created for you on config import) 

- In that menu add five links and place in the following order:
&nbsp;&nbsp;&nbsp;&nbsp; 1. Menu link title: All topics and services
&nbsp;&nbsp;&nbsp;&nbsp; -- Link: /#all-topics-header
&nbsp;&nbsp;&nbsp;&nbsp; -- Description: [empty]
&nbsp;&nbsp;&nbsp;&nbsp; 2. Menu link title: About the U.S. and its government
&nbsp;&nbsp;&nbsp;&nbsp; -- Link: /about-the-us
&nbsp;&nbsp;&nbsp;&nbsp; -- Description: usa-nav__about
&nbsp;&nbsp;&nbsp;&nbsp; 3. Menu link title: Money
&nbsp;&nbsp;&nbsp;&nbsp; -- Link: /money
&nbsp;&nbsp;&nbsp;&nbsp; -- Description: usa-nav__money
&nbsp;&nbsp;&nbsp;&nbsp; 4. Menu link title: Laws and legal issues
&nbsp;&nbsp;&nbsp;&nbsp; -- Link: /laws-and-legal-issues
&nbsp;&nbsp;&nbsp;&nbsp; -- Description: usa-nav__law
&nbsp;&nbsp;&nbsp;&nbsp; 5. Menu link title: Scams and fraud
&nbsp;&nbsp;&nbsp;&nbsp; -- Link: /scams-and-fraud
&nbsp;&nbsp;&nbsp;&nbsp; -- Description: usa-nav__scams

- Do the same for the Spanish menu:
&nbsp;&nbsp;&nbsp;&nbsp; 1. Menu link title: Todos los temas y servicios
&nbsp;&nbsp;&nbsp;&nbsp; -- Link:/es#all_topics
&nbsp;&nbsp;&nbsp;&nbsp; -- Description: [empty]
&nbsp;&nbsp;&nbsp;&nbsp; 2. Menu link title: Acerca de EE. UU. y directorios del Gobierno
&nbsp;&nbsp;&nbsp;&nbsp; -- Link: /es/acerca-de-estados-unidos
&nbsp;&nbsp;&nbsp;&nbsp; -- Description: usa-nav__acerca
&nbsp;&nbsp;&nbsp;&nbsp; 3. Menu link title: Dinero
&nbsp;&nbsp;&nbsp;&nbsp; -- Link: /es/dinero
&nbsp;&nbsp;&nbsp;&nbsp; -- Description: usa-nav__dinero
&nbsp;&nbsp;&nbsp;&nbsp; 4. Menu link title: Leyes y asuntos legales
&nbsp;&nbsp;&nbsp;&nbsp; -- Link: /es/leyes-y-asuntos-legales
&nbsp;&nbsp;&nbsp;&nbsp; -- Description: usa-nav__leyes
&nbsp;&nbsp;&nbsp;&nbsp; 5. Menu link title: Estafas y fraudes
&nbsp;&nbsp;&nbsp;&nbsp; -- Link: /es/estafas-y-fraudes
&nbsp;&nbsp;&nbsp;&nbsp; -- Description:  usa-nav__estafas

- Refresh the cache.
- The description is used as the id for the list item in the template so pay attention that it is copied correctly (double underscores) 
- Menu should appear as it used to

## Checklist for the Developer



## Checklist for the Peer Reviewers
